### ISSUE TITLE
down_ratio

### ISSUE THREAD

**Issue by wolfworld6** [yuqj1990/caffe_train#11]
when i run the inference code，i got this error，and DetectionOutputParameter has no field named "down_ratio" on the caffe.proto
Net('/home/training/disk/dxd/dxd/caffe_train/examples/face/detector/prototxt/Full_640x640/CenterGridSoftmax_face_v4_deploy.prototxt', 1, weights='/home/training/disk/dxd/dxd/caffe_train/examples/face/detector/prototxt/snapshot/CenterGridSoftmax_face_v4_0_iter_56000.caffemodel')
[libprotobuf ERROR google/protobuf/text_format.cc:274] Error parsing text-format caffe.NetParameter: 4238:15: Message type "caffe.DetectionOutputParameter" has no field named "down_ratio".

**Comment by yuqj1990** [comment]
you should pull code again,and re-complie, because i add 320x320 face detection version,

**Comment by wolfworld6** [comment]
> you should pull code again,and re-complie, because i add 320x320 face detection version,

Have you tested your model ，is it right？

**Comment by yuqj1990** [comment]
of course

**Comment by wolfworld6** [comment]
> of course

i test my model,it is not good,the loss as follows:
![image](https://user-images.githubusercontent.com/18051979/85271924-0a663100-b4ae-11ea-9676-6d600fcb249b.png)

**Comment by yuqj1990** [comment]
your task is face detection? or other else? you can modify the anchor scales depend on your data-set distribution, and the loss value while i end the training is about 30-40, because i use batch_size as the normalize , you also can modify the type by using the valid positive bboxes num by set the normalization as  VALID

**Comment by yuqj1990** [comment]
and, if you evaluate your performance about your model, you can use the confs_threshold 0.05 like i used in the prototxt, but if you test on the realtime, you can raise the value and the nms_threshold, and the top_K value

**Comment by wolfworld6** [comment]
> your task is face detection? or other else? you can modify the anchor scales depend on your data-set distribution, and the loss value while i end the training is about 30-40, because i use batch_size as the normalize , you also can modify the type by using the valid positive bboxes num by set the normalization as VALID

yeah，i just use your deploy ，have not changed any param，and the training dataset is wider_face

**Comment by yuqj1990** [comment]
all right, the wider_face you used is from retina_wider_face modified or original from the official web-page? and ...finally, if everything is ok and same with me, i dont know the reason. just like i said if you evaluated the performance, you can keep the same with me , but if you want to test in the camera ,you should make the confidence, nms, and top_K threshold more higher. and you can leave you email , i will share my weights file

**Comment by wolfworld6** [comment]
> higher
ok，thx，15011232850@163.com
